vhdbuilder/packer/vhd-scanning-exe-on-vm.sh

```diff
@@ -23,7 +23,13 @@ elif [[ "$OS_SKU" == "Ubuntu" ]] && [[ "$OS_VERSION" == "22.04" ]] && [[ "${ARCH
         echo "Azure CLI is not installed properly."
         exit 1
     fi
-elif [[ "$OS_SKU" == "Ubuntu" ]] && [[ "$OS_VERSION" == "22.04" ]] && [[ "${ARCHITECTURE,,}" != "arm64" ]]; then
+elif [[ "$OS_SKU" == "Ubuntu" ]] && [[ "$OS_VERSION" == "24.04" ]] && [[ "${ARCHITECTURE,,}" == "arm64" ]]; then
```

Comment:
wish we could use a switch for all these,  this is screaming for a go refactor...

Reply:
Yeah I agree. Hopefully I can take some time to look at it after NIC work